core/keyserver/keyserver.go

```diff
@@ -85,7 +84,12 @@ func New(logID int64,
 // this user and that it is the same one being provided to everyone else.
 // GetEntry also supports querying past values by setting the epoch field.
 func (s *Server) GetEntry(ctx context.Context, in *tpb.GetEntryRequest) (*tpb.GetEntryResponse, error) {
-	return s.getEntry(ctx, in.UserId, in.AppId, in.FirstTreeSize, -1)
+	resp, err := s.getEntry(ctx, in.UserId, in.AppId, in.FirstTreeSize, -1)
```

Comment:
This should be fine the way it was, `getEntry` already logs the error and returns a gRPC error with not much information. I don't think there's a need to log that.